CHANGELOG.md

```diff
@@ -18,6 +18,11 @@ Main (unreleased)
 - Grafana Agent Flow: Add tracing instrumentation and a `tracing` block to
   forward traces to `otelcol` component. (@rfratto)
 
+- New Grafana Agent Flow components:
+
+  - `remote.http` polls an HTTP URL and exposes the response body as a string
+    or secret to other components. (@rfratto)
```

Comment:
Let's not forget to register the component on `all.go`

Reply:
🤦‍♂️ thanks, I can't believe I'm still forgetting to do that :)